server/src/controllers/proposal.controller.js

```diff
@@ -29,22 +29,24 @@ export const getLevels = async (req, res, next) => {
 
 export const postProposal = async (req, res) => {
   try {
-    const title = req.body.title;
-    const type = req.body.type;
-    const description = req.body.description;
+    const title = req.body.title.trim();
```

Comment:
In this file the green part can be accepted because I think that are the updates made by you or Oscar